develop/security-best-practices.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Security best practices
+description: Image security best practices guide
+keywords: docker, images, containers, vulnerability, cve
+redirect_from:
+  - /develop/scan-images/
+---
+
+As a developer, you can take a few simple steps to improve the security of your
```

Comment:
@dvdksn I'd possibly say you don't need to say "as a developer" as it's a bit exclusionary, but maybe there's a reason it's used? I'd also pretty much always remove words like "simple".

```suggestion
You can take a few steps to improve the security of your
```

Reply:
Yes, both suggestions are perfect! Thank you!!